apps/src/javalab/CommitDialog.jsx

```diff
@@ -24,7 +58,10 @@ export class UnconnectedCommitDialog extends React.Component {
     let commitText = i18n.commit();
     const saveInProgress =
       this.state.backpackSaveInProgress || this.state.commitSaveInProgress;
-    const isCommitButtonDisabled = !this.state.commitNotes || saveInProgress;
+    const isCommitButtonDisabled =
```

Comment:
clean up nit: we reference `this.state` a lot in this function we could instead do
```
const {commitNotes, hasBackpackLoadError, ...} = this.state
```
and then we wouldn't need to reference variables with the `this.state` prefix